packages/core/src/components/StepList/StepList.jsx

```diff
@@ -20,7 +20,9 @@ export const StepList = ({ steps, ...props }) => (
 export const stepShape = {
   id: PropTypes.string,
   href: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired, // [Deprecated]
+  heading: PropTypes.string.isRequired,
+  headingLevel: PropTypes.oneOf(['1', '2', '3', '4', '5']),
```

Comment:
`title` should be replaced with `heading` for the following prop descriptions
- steps
- actionsLabelText
- descriptionLabelText
- substepsLabelText

Reply:
I cant remove the `title` in steps if its to be backwards compatible.